source/codegen/metadata/nidigitalpattern/enums.py

```diff
@@ -113,6 +112,19 @@
             }
         ]
     },
+    'HistoryRamCyclesToAcquire': {
```

Comment:
This is not only out of ABC order, it also has a different RAM capitalization.

Reply:
Oh that's actually the reason, since the sorting is case sensitive.

Reply:
It's still a different case, Ram vs RAM.

Reply:
Yeah, I think it's that the others changed to be RAM instead of Ram. So in grpc_device/enums/enums.hapigen can you add 'name' overrides for 'history ram max samples to acquire per site' and 'history ram trigger type' like you did for 'history ram cycles to acquire'.